packages/datadog-plugin-graphql/src/utils.js

```diff
@@ -0,0 +1,40 @@
+function extractErrorIntoSpanEvent (config, span, exc) {
+  const attributes = {}
+
+  if (exc.name) {
+    attributes.type = exc.name
+  }
+
+  if (exc.stack) {
+    attributes.stacktrace = exc.stack
+  }
+
+  if (exc.locations) {
+    attributes.locations = []
+    for (const location of exc.locations) {
+      attributes.locations.push(`${location.line}:${location.column}`)
+    }
+  }
+
+  if (exc.path) {
+    attributes.path = exc.path
```

Comment:
We need to stringify the elements in `exc.path` because span events attributes do not support heterogeneous arrays (arrays with a mix of integers and strings), and `exc.path` can be `["query", 0, "id]` when there are arrays in the GraphQL path.

Reply:
Okay, I'll fix that 👍